src/types/index.ts

```diff
@@ -11,7 +11,9 @@ export const DEFAULT_SETTINGS: SettingsProp = {
 };
 
 export interface ContentProp {
+	id?: string;
 	title: string;
+	updated_at?: string;
```

Comment:
In the version I'm rewriting, the properties are `post-id` and `post-last-updated`. These need to be consistent so that users who upgrade don't have to migrate their notes.

```suggestion
	post-id?: string;
	title: string;
	post-last-updated?: string;
```